common/PersistenceFormat.js

```diff
@@ -14,8 +14,9 @@ module.exports = {
         return path.join(config.get('rootFolderPath'), entryId);
     },
 
-    getEntryRelativePath: function (entryId) {
-        return entryId;
+    getEntryBasePath: function (path) {
+        let subString = path.match(/([01]_\w+\/[0-9]+\/)/)[1];
+        return path.substr(0, path.lastIndexOf(subString) + subString.length);
     },
 
```

Comment:
When PlaylistGenerator creates a new ConcatSource instance it send as a parameter the path for the folder where to store the files (retrieved from fs-utils):
/web/content/kLive/live/<entry_Id>/<day>/<hour>/ 
Since the ConcatSource requires only the BASE path I need to attract it from the full file path. That way the playlist.json would show as basePath: /web/content/kLive/live/<entry_Id>/
and each file specific path would be: <day>/<hour>/<file_name>/

Why do I need to modify the regEx? it works fine